cmd/release-notes/main.go

```diff
@@ -206,6 +206,13 @@ func init() {
 		util.EnvDefault("RELEASE_TARS", ""),
 		"Directory of tars to sha512 sum for display",
 	)
+
+	cmd.PersistentFlags().BoolVar(
+		&opts.TableOfContents,
+		"toc",
+		util.IsEnvSet("TOC"),
```

Comment:
Looks like we are defaulting to `false` is the env var is not set. Do you think it more common that we do or do not want a TOC?

Reply:
Good question, my suggestion was to leave the TOC out for the manual generations, which we do on a regular cadence for minor releases. I think we could stick with that for now :)

Reply:
That sounds reasonable to me!